backend/urls.py

```diff
@@ -44,21 +42,19 @@
 ]
 
 # This is to aid local development. 
-# This exposes the frontend/public/plasmic directory on a url in static and allows browsing.
+# This exposes the frontend/public directory on a url in static and allows browsing.
 # Whitenoise currently handles this otherwise
 if settings.DEBUG:
     from django.contrib.staticfiles import views as static_views
 
     urlpatterns += [
-        re_path(r'^static/plasmic/(?P<path>.*)$', 
+        re_path(r'^static/(?P<path>.*)$',
             view = static.serve, 
             kwargs = {
-                'document_root': os.path.join(settings.BASE_DIR, 'frontend/public/plasmic'), 
+                'document_root': os.path.join(settings.BASE_DIR, 'frontend/public'),
                 'show_indexes' : True
             }
         ),
         # Matches STATIC_URL in settings
         re_path(r'^django_static/(?P<path>.*)$', static_views.serve),
```

Comment:
@jonespm, can you help explain why I still see `plasmic` as a directory if I go to browse the static via the browser? Maybe it's a config thing, but I haven't figured it out yet.